config/src/main/java/org/hyperledger/besu/config/StubGenesisConfigOptions.java

```diff
@@ -467,17 +467,17 @@ public List<Long> getForkBlockTimestamps() {
 
   @Override
   public Optional<Address> getWithdrawalRequestContractAddress() {
-    return Optional.empty();
+    return Optional.of(Address.fromHexString("0x00000961ef480eb55e80d19ad83579a64c007002"));
```

Comment:
what do you think about a MainnetGenesisConfigOptions subclass (or interface with default implementations) for these methods?  StubGenesis seems to be used everywhere, not just for mainnet PoS networks

Reply:
I like the idea, but i'm not sure where to draw the boundaries - would a stub genesis only contain frontier features? everything up to the DAO fork? everything up to The Merge?

Perhaps only things used by all PoW, PoS, and PoA networks?